examples/kubernetes/1.10/cilium-ds.yaml

```diff
@@ -51,7 +51,7 @@ spec:
                   optional: true
                   key: clean-cilium-state
       containers:
-        - image: docker.io/cilium/cilium:latest
+        - image: docker.io/cilium/cilium:v1.2.0-rc1
           imagePullPolicy: Always
```

Comment:
`ImagePullPolicy: IfNotPresent` for all k8s versions

Reply:
Thanks for catching this, I somehow reversed the two in my head when making the release on Friday.